Sources/UIPanGestureRecognizer.swift

```diff
@@ -63,10 +63,10 @@ open class UIPanGestureRecognizer: UIGestureRecognizer {
     open override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent) {
         super.touchesBegan(touches, with: event)
         guard let trackedTouch = touches.first else { return }
-        state = .began
         self.trackedTouch = trackedTouch
         initialTouchPoint = trackedTouch.location(in: nil)
         touchesMovedTimestamp = trackedTouch.timestamp
+        state = .began
```

Comment:
this is quite important, otherwise `self.trackedTouch` is still `nil` when touchesBegan is executed